tests/Tutor.Web.Tests/Integration/Learning/Utilities/FeedbackTests.cs

```diff
@@ -23,14 +23,15 @@ public void Stores_emotions_feedback(EmotionsFeedbackDto postedFeedback, Emotion
         using var scope = Factory.Services.CreateScope();
         var controller = SetupFeedbackController(scope);
         var dbContext = scope.ServiceProvider.GetRequiredService<TutorContext>();
+        dbContext.Database.BeginTransaction();
```

Comment:
We should write down the guidelines this list https://app.clickup.com/24556453/v/l/li/900400270031 for using UoW in the context of production features and in the context of tests.